### PR TITLE
Prow service pods metrics scraped by Google Managed Prometheus

### PR DESCRIPTION
**Pull request by chaodaiG**
Previously adopted GKE workload metrics solution was superceded by [GMP](https://cloud.google.com/stackdriver/docs/solutions/gke/gmp-migration), migrating to GMP by following the instruction

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
do we need to setup GMP before we merge? or is it already

**Comment by chaodaiG** [comment]
we will need to enable GMP on prow cluster from inside istio-testing project by following https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#config-mgd-collection, it would be a simple toggle on the UI if it's meant to be done manually. @stewartbutler and/or @howardjohn , is there an automated process to do so?

**Comment by howardjohn** [comment]
We are working on IaC but not yet. So we can press the checkbox

**Comment by chaodaiG** [comment]
> We are working on IaC but not yet. So we can press the checkbox

cool, done

**Review by fejta — APPROVED** [review]
/hold

**Review by cjwagner — APPROVED** [review]
/unhold

**Pull request merged**
